[2024-01-30T17:38:56.322+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_api_dag.get_weather_data scheduled__2024-01-30T09:00:00+00:00 [queued]>
[2024-01-30T17:38:56.394+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_api_dag.get_weather_data scheduled__2024-01-30T09:00:00+00:00 [queued]>
[2024-01-30T17:38:56.396+0000] {taskinstance.py:2171} INFO - Starting attempt 1 of 1
[2024-01-30T17:38:56.500+0000] {taskinstance.py:2192} INFO - Executing <Task(PythonOperator): get_weather_data> on 2024-01-30 09:00:00+00:00
[2024-01-30T17:38:56.508+0000] {standard_task_runner.py:60} INFO - Started process 3003 to run task
[2024-01-30T17:38:56.521+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'weather_api_dag', 'get_weather_data', 'scheduled__2024-01-30T09:00:00+00:00', '--job-id', '88', '--raw', '--subdir', 'DAGS_FOLDER/weather_api_dag.py', '--cfg-path', '/tmp/tmpz_39mqe0']
[2024-01-30T17:38:56.528+0000] {standard_task_runner.py:88} INFO - Job 88: Subtask get_weather_data
[2024-01-30T17:38:56.668+0000] {task_command.py:423} INFO - Running <TaskInstance: weather_api_dag.get_weather_data scheduled__2024-01-30T09:00:00+00:00 [running]> on host 68c8327b56d3
[2024-01-30T17:38:56.956+0000] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_api_dag' AIRFLOW_CTX_TASK_ID='get_weather_data' AIRFLOW_CTX_EXECUTION_DATE='2024-01-30T09:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-30T09:00:00+00:00'
[2024-01-30T17:38:57.640+0000] {logging_mixin.py:188} INFO - {'cod': 401, 'message': 'Invalid API key. Please see https://openweathermap.org/faq#error401 for more info.'}
[2024-01-30T17:39:02.043+0000] {logging_mixin.py:188} INFO - {'record': {'cod': 401, 'message': 'Invalid API key. Please see https://openweathermap.org/faq#error401 for more info.'}, 'metadata': {'id': '65b934361f5677401f2873c5', 'createdAt': '2024-01-30T17:39:02.065Z', 'private': True, 'name': 'weather_07438527-7659-490a-88fb-bb0115fbc116', 'collectionId': '65ab4466266cfc3fde7c6f31'}}
[2024-01-30T17:39:02.050+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-01-30T17:39:02.077+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=weather_api_dag, task_id=get_weather_data, execution_date=20240130T090000, start_date=20240130T173856, end_date=20240130T173902
[2024-01-30T17:39:02.123+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-01-30T17:39:02.178+0000] {taskinstance.py:3281} INFO - 0 downstream tasks scheduled from follow-on schedule check
